src/main/resources/lode/swrl-module.xsl

```diff
@@ -52,7 +52,8 @@ Copyright (C) 2023, Victor Chavez <vchavezb@protonmail.com>
     <xsl:template name="get.swrl.toc">
         <ul class="hlist">
             <xsl:apply-templates select="/rdf:RDF/rdf:Description[rdf:type[@rdf:resource='http://www.w3.org/2003/11/swrl#Imp']]" mode="toc">
-                <xsl:sort select="rdfs:comment" order="ascending" data-type="text"/>
+                <xsl:sort select="lower-case(f:getLabel(rdfs:label))"
+                          order="ascending" data-type="text"/>
```

Comment:
this triggers an error if the label does not exist

Reply:
I assumed all the swrl rules had an rdfs:label because in the SWRL Tab from Protégé the rdfs: label is always included.


![grafik](https://github.com/dgarijo/Widoco/assets/47216966/8cea37e8-11a0-4bbd-8fc7-7371877327f8)

I am not sure if this will work but we could perhaps check if the rdfs:label exists 

```xml
<xsl:choose>
    <xsl:when test="exists(rdfs:label)">
        <xsl:sort select="lower-case(f:getLabel(rdfs:label))" order="ascending" data-type="text"/>
    </xsl:when>
    <xsl:when test="exists(rdfs:comment)">
        <xsl:sort select="lower-case(f:getLabel(rdfs:comment))" order="ascending" data-type="text"/>
    </xsl:when>
</xsl:choose>
```